spec/helpers/application_helper_spec.rb

```diff
@@ -418,6 +418,15 @@ def preload_link(url)
   end
 
   describe 'crawlable_meta_data' do
+
+    it 'Supports ASCII URLs with odd chars' do
+      result = helper.crawlable_meta_data(
+        url: -((+"http://localhost/ión").force_encoding("ASCII-8BIT"))
```

Comment:
Just a tiny bit less magic? 😉 

```suggestion
        url: (+"http://localhost/ión").force_encoding("ASCII-8BIT").freeze
```